packages/shylu/shylu_node/basker/example/Solve.cpp

```diff
@@ -75,17 +66,15 @@ int main(int argc, char* argv[])
   //Teuchos::Time::Time ts("stime",true);
   solver->symbolicFactorization();
   //ts.stop();
-  //cout << "After SFactor, Time : " 
+  //cout << "After SFactor, Time : "
   //     << ts.totalElapsedTime() << endl;
   cout << "Before Numerical Factorization" << endl;
   //Teuchos::Time::Time t("time", true);
   solver->numericFactorization();
   //t.stop();
-  //cout << "After Numerical Factorization, Time: " 
+  //cout << "After Numerical Factorization, Time: "
   //     << t.totalElapsedTime() << endl;
   solver->solve();
 
-  Kokkos::finalize();
```

Comment:
Same as above. No need for finalize ?

Reply:
Same as above; done when ScopeGuard goes out of scope.